com.visualstudio.code.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.visualstudio.code.desktop</id>
+	<metadata_license>Multiple, see https://code.visualstudio.com/license</metadata_license>
```

Comment:
This should be the same syntax as `project_license`.

Reply:
One problem though is this license would then not be distributable, but.. the same appdata file is under a different license if the foss version is built...

upstream really needs to fix this.

Reply:
Right, I'm going to update the metadata to be under the MIT, which is where I've 'forked' it from for these purposes.